cmd/cloudshell_open/cloudrun_test.go

```diff
@@ -29,16 +29,18 @@ func Test_tryFixServiceName(t *testing.T) {
 		{"trailing dash", "abc-def-", "abc-def"},
 		{"trailing dashes", "abc-def---", "abc-def"},
 		{"only dashes, trimmed and prefixed", "---", "svc"},
+		{"upper to lower", "Foo", "foo"},
 		{"truncate to 63 chars",
 			"A123456789012345678901234567890123456789012345678901234567890123456789",
-			"A12345678901234567890123456789012345678901234567890123456789012"},
+			"a12345678901234567890123456789012345678901234567890123456789012"},
 		{"already at max length - no truncate",
 			"A12345678901234567890123456789012345678901234567890123456789012",
-			"A12345678901234567890123456789012345678901234567890123456789012",
+			"a12345678901234567890123456789012345678901234567890123456789012",
 		},
 		{"leading digit + trunc",
 			"0123456789012345678901234567890123456789012345678901234567890123456789",
 			"svc-01234567890123456789012345678901234567890123456789012345678"},
+		{"spaces", "foo bar", "foo-bar"},
```

Comment:
can you add invalid chars to the beginning? also maybe generalize this as: `invalid chars` and use `"hello, world"` as the value. that actually tests `hello-world` and not `hello--world` (due to `+` in regexp)